src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    private static final int INDEX_OPERATION = 0;
+    private static final int INDEX_FRUIT = 1;
+    private static final int INDEX_QUANTITY = 2;
+    private static final String HEADER = "type,fruit,quantity";
+    private static final String DELIMITER = ",";
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : inputReport) {
+            if (line.equals(HEADER)) {

```

Comment:
Using a hardcoded header string to skip the first line is not flexible. Consider a more robust way to handle the header, such as passing a boolean parameter to indicate whether to skip the header or not.